lib/sparql.jison

```diff
@@ -771,23 +771,23 @@ PathProperties
     ;
 
 PathProperty
-    : 'START' Var '='? PathValue 
+    : 'START' Var '='? PathValue?
```

Comment:
Does the jison syntax allow grouping? If there is a PathValue then there must be an equals. So something like `( = PathValue )?`

Reply:
I thought PathValue should always be present? In that case I can remove the optional for =.

Reply:
Only the variable is required, not the optional assignment to a starting value or pattern.